tower-http/src/trace/on_body_chunk.rs

```diff
@@ -8,25 +9,28 @@ pub trait OnBodyChunk<B> {
     ///
     /// `latency` is the duration since the response was sent or since the last body chunk as sent.
     ///
+    /// `span` can be used to record field values that weren't known when the span was
+    /// created.
```

Comment:
what do you think about expanding on this a little bit:
```suggestion
    /// `span` is the `tracing` [`Span`] corresponding to this request, produced
    /// the closure passed to [`TraceLayer::make_span_with`]. It can be used to
    /// [record field values][record] that weren't known when the span was
    /// created.
    ///
    /// [`Span`]: https://docs.rs/tracing/latest/tracing/span/index.html
    /// [record]: https://docs.rs/tracing/latest/tracing/span/struct.Span.html#method.record
```

Reply:
I like it!